spec/factories/user.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+
+  factory :user do
+    email 'test@example.com'
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping. (https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)